### PR TITLE
Add +/- to expand collapse on accordion

### LINKED ISSUE #1397: Add icons and buttons to make accordion functionality more clear to user
## Action Requested
to be added

## Acceptance Criteria
to be added

## Testing
to be added

## Background & Context
The expand/collapse functionality of the eCR Document should be more clear to the user with common iconography, and they should have the option to easily collapse or expand all with just one click

## Designs
[Figma Designs](https://www.figma.com/file/tXWUeV3Kjehd7VNqHojZYq/eCR-Viewer?type=design&node-id=1024%3A136062&mode=design&t=lClwL7CnvrviunUF-1)
[Figma Clickable Prototype](https://www.figma.com/proto/tXWUeV3Kjehd7VNqHojZYq/eCR-Viewer?page-id=871%3A2283&type=design&node-id=1606-27619&viewport=1011%2C558%2C0.05&t=sGUJKIN0MXVFYIfm-1&scaling=min-zoom&starting-point-node-id=1606%3A27619&mode=design)

add buttons to expand/collapse at the top:

icons should change depending on state:

## Design Review
what to look for for design review:
- check for capitalization on the buttons
- check color on buttons (should be primary, hex: 005EA2)
- check for spacing and alignment
- make sure the icons are + signs when collapsed and - signs when expanded
- clickable area should be the whole teal box
- useful to see a video of the functionality to ensure the right thing happens when you click collapse/expand all buttons and interaction looks okay (nothing jarring/glitchy)

## Additional Information
to be added

### PR DESCRIPTION
**Pull request by lina-roth**
# PULL REQUEST

### Add +/- to expand collapse on accordion

https://github.com/CDCgov/phdi/assets/21372324/41b8f0d3-df18-42b2-b12a-92a6bb67d1f7

https://github.com/CDCgov/phdi/assets/21372324/da7e1b82-17fc-4694-bfa5-1c5dbe9bfcd4

### Add Expand All and Collapse all to Ecr Document and Lab info sections 

https://github.com/CDCgov/phdi/assets/21372324/e7bfdbe2-c27f-4405-9bfd-eb64f250d021

https://github.com/CDCgov/phdi/assets/21372324/32986151-8c77-48a3-9f08-8a05da5e06a1

## Related Issue
Fixes #1397 

## Additional Information
Anything else the review team should know?

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

**Comment by maryyeh** [comment]
@lina-roth - thanks for the videos, super helpful, looks good!

One question, in the videos, it seemed like you're mostly clicking on the icon, in the design review area, it says the whole teal box should be clickable for the expand/collapse, is that the case? If yes, then this is approved!

**Comment by lina-roth** [comment]
@maryyeh sorry I should've made that more clear in the video. Yes the whole header is clickable. 

https://github.com/CDCgov/phdi/assets/21372324/da85450d-b898-46fe-ae6e-c2cd6fccab0b

**Comment by maryyeh** [comment]
@lina-roth  - thanks for the additional video. Approved!

File: containers/ecr-viewer/src/app/view-data/components/tests/EcrMetadata.test.tsx
Comment (BobanL) [inline containers/ecr-viewer/src/app/view-data/components/tests/EcrMetadata.test.tsx:1]: This change was made because the tests here were incorrect.

**Review by BobanL — APPROVED** [review]

File: containers/ecr-viewer/src/app/view-data/components/tests/EcrMetadata.test.tsx
Comment (JNygaard-Skylight) [reply containers/ecr-viewer/src/app/view-data/components/tests/EcrMetadata.test.tsx:1]: Thanks for fixing my tests 😬

**Review by JNygaard-Skylight — APPROVED** [review]
LGTM

**Pull request merged**